### PR TITLE
Add missing env var to config map

### PR DESCRIPTION
**Pull request by JustinCast**
### GH Issue

#

### Steps to test

1.
1.
1.

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [ ] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [ ] I have taken into consideration any impact to site performance.

### CODE REVIEW

**Review by kuronosec — APPROVED** [review]
LGTM

File: services/frontend/src/routes/block-producers/block-producer-rate.js [hunk 1]
```diff
@@ -186,6 +186,7 @@ const BlockProducerRate = ({ account, ual }) => {
         broadcast: true
       })
 
+      console.log('before mutation')
```
Comment (xavier506) [inline services/frontend/src/routes/block-producers/block-producer-rate.js:189]: do we need these console logs?

**Comment by xavier506** [comment]
@JustinCast  was there and issue with port 3005?

also  please review conflicts

**Comment by JustinCast** [comment]
> @JustinCast was there and issue with port 3005?
> 
> also please review conflicts

@xavier506 I think that the problem of rating has to with hasura not able to find the right webhook port redirection

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**